### ISSUE TITLE
Discrepancy between AUC score and ROC plot

### ISSUE THREAD

**Issue by SherlockMones** [frankkramer-lab/aucmedi#165]
Testing landmark classification, that results in bad AUC scores (between 0.4 and 0.6), although ROC shows impeccable classification. F1 also amazing scores (~ 0.9). See screeenshots for details

![image](https://user-images.githubusercontent.com/42068285/181219541-f0047efe-3be2-4937-99ed-4b45f9027ec6.png)

![image](https://user-images.githubusercontent.com/42068285/181219701-2d124a4c-ad10-4a07-b579-1a3dafdca290.png)

**Comment by muellerdo** [comment]
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html

**Comment by muellerdo** [comment]
Before:
![plot performance barplot](https://user-images.githubusercontent.com/28828967/181241829-b269ca36-4ade-4276-aaf7-63ad4939462e.png)

After:
![plot performance barplot](https://user-images.githubusercontent.com/28828967/181241870-955db892-3cf6-469a-a699-e3bf5caf2620.png)

**Comment by muellerdo** [comment]
Issue was incorrect probability access if no threshold is provided (automatic). For some logic error I thought I had to pick the maximum softmax of all classes which is obviously incorrect...

Fixed it! :tada:

**Comment by muellerdo** [comment]
PR started

**Comment by muellerdo** [comment]
Merged in dev branch.
